ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/WearApp.kt

```diff
@@ -505,6 +514,13 @@ fun WearApp(
     }
 }
 
+@Composable
+private fun menuNameAndCallback(
```

Comment:
TIOLI: Why not just a simple data class with resource id, and screen.  Leave navigate and stringResource to the LandingScreen?